<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SOS_settings_Dialog</class>
 <widget class="QDialog" name="SOS_settings_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="familyNum_lineEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>44</y>
     <width>119</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="friendsNum_lineEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>124</y>
     <width>119</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="colleaguesNum_lineEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>204</y>
     <width>119</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="famNum_pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="friNum_pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colNum_pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1_120406110829_5.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>21</y>
     <width>42</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>family</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>101</y>
     <width>46</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>friends</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>181</y>
     <width>72</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>colleagues</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>familyNum_lineEdit</zorder>
  <zorder>friendsNum_lineEdit</zorder>
  <zorder>colleaguesNum_lineEdit</zorder>
  <zorder>famNum_pushButton</zorder>
  <zorder>friNum_pushButton</zorder>
  <zorder>colNum_pushButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
